src/test/java/nextstep/subway/section/SectionAcceptanceTest.java

```diff
@@ -173,4 +290,13 @@ public void setUp() {
                                     .flatMap(station -> Stream.of(station.getId()))
                                     .collect(Collectors.toList());
     }
+
+    public static ExtractableResponse<Response> 구간을_제거한다(Long lineId, Long stationId) {
+        return RestAssured.given().log().all()
+                .body(stationId)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().delete("/lines/" + lineId + "/sections")
```

Comment:
```suggestion
                .when().delete("/lines/{lineId}/sections", lineId)
```

다음과 같이 작성한다면 조금 더 친절한 리포트를 볼 수 있습니다 :)